gallery/workbook/Azure Advisor.json

```diff
@@ -17,6 +17,13 @@
 					"author": "Microsoft",
 					"name": "Service Retirement (Preview)",
 					"description": "Track service retirements for your Azure resources"
+				},
+				{
+					"id": "Workbooks/Azure Advisor/Cost Optimization",
+					"author": "Microsoft",
+					"name": "Cost Optimization (Preview)",
+					"description": "This workbook surfaces key recommendations that can directly impact cost savings. These recommendations include identifying and addressing idle resources, leveraging Azure Advisor recommendations such as Reserved Instances and Savings Plans for Compute, and optimizing the usage of oversized VMs. ",
```

Comment:
can the description be shortened, only a few words would be visible in the UI

Reply:
shortened.